piperider_cli/assertion_engine/assertion.py

```diff
@@ -168,7 +169,10 @@ def __init__(self, profiler, assertion_search_path=PIPERIDER_ASSERTION_SEARCH_PA
         self.assertion_search_path = assertion_search_path
         self.assertions_content = {}
         self.assertions = []
-        pass
+
+        self.default_plugins_dir = os.path.abspath(os.path.join(self.assertion_search_path, '../plugins'))
```

Comment:
We can just hard code the `default_plugin_dir` by using `PIPERIDER_ASSERTION_PLUGIN_PATH =  os.path.join(os.getcwd(), PIPERIDER_WORKSPACE_NAME, 'plugin')`
The assertion_search_path can be assigned in the constructor of AssertionEngine class.

Reply:
got it, let me rebase the branch to get the new variables